From f3ad342636c31002952935667108b63a38f1f0ca Mon Sep 17 00:00:00 2001
From: Peter Harliman Liem <peter.harliman.liem@intel.com>
Date: Thu, 15 Aug 2019 20:14:31 +0800
Subject: [PATCH] DRVLIB_SW-2399 - Add gphy PEF7072 into lantiq phy driver

This gphy is used as external wan phy in Spear Point board.
We utilize same handlers as PEF7071 as there are no need
to implement new ones.
---
 drivers/net/phy/lantiq.c | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/drivers/net/phy/lantiq.c b/drivers/net/phy/lantiq.c
index e263c6815c8b..45644c970905 100644
--- a/drivers/net/phy/lantiq.c
+++ b/drivers/net/phy/lantiq.c
@@ -270,6 +270,16 @@ static struct phy_driver lantiq_phy[] = {
 		.ack_interrupt	= vr9_gphy_ack_interrupt,
 		.did_interrupt	= vr9_gphy_did_interrupt,
 		.config_intr	= vr9_gphy_config_intr,
+	}, {
+		.phy_id		= 0xd565a401,
+		.phy_id_mask	= 0xffffffff,
+		.name		= "Intel GPHY PEF7072",
+		.features	= (PHY_GBIT_FEATURES | SUPPORTED_Pause),
+		.config_init	= vr9_gphy_config_init,
+		.config_aneg	= vr9_gphy_config_aneg,
+		.read_status	= genphy_read_status,
+		.ack_interrupt	= vr9_gphy_ack_interrupt,
+		.did_interrupt	= vr9_gphy_did_interrupt,
 	}, {
 		.phy_id		= 0x030260D0,
 		.phy_id_mask	= 0xfffffff0,
